extensions/amp-story/1.0/bookend/bookend-component.js

```diff
@@ -160,13 +161,18 @@ export class BookendComponent {
 
     components = prependTitle(components, localizationService);
 
+    const data = {position: 1};
     components.forEach(component => {
       const {type} = component;
       if (type && componentBuilderInstanceFor(type)) {
-        fragment.appendChild(
-          componentBuilderInstanceFor(type).buildElement(component, doc)
+        const el = componentBuilderInstanceFor(type).buildElement(
+          component,
+          doc,
+          data
```

Comment:
The position is unreliable here, since `prependTitle` could potentially add a component, offsetting everything by one.
We could replace this `data` with `{position: index}` where `index` comes from the `forEach`